src/Helpers/useUrlParams.test.js

```diff
@@ -0,0 +1,35 @@
+import { renderHook, act } from '@testing-library/react';
+import { createTestWrapper } from '../Utilities/TestWrapper';
+import { MemoryRouter } from 'react-router-dom';
+
+import useUrlParams, { constructURLParameters } from './useUrlParams';
+
+describe('MiscHelper', () => {
+    it('useUrlParams', async () => {
+        const { result } = renderHook(() =>  useUrlParams(['a', 'b']), {
+            wrapper: createTestWrapper(MemoryRouter, { initialEntries: ['/'] })
```

Comment:
Can we use the [TestWrapper](https://github.com/RedHatInsights/frontend-components/blob/1599246a16e70d78401e70628797d5fef24f4fae/packages/utils/src/TestingUtils/JestUtils/TestWrapper.js) which has already MemortRouter provided?